eth/protocol_test.go

```diff
@@ -230,12 +230,12 @@ func TestForkIDSplit(t *testing.T) {
 	peerNoFork = newPeer(64, p2p.NewPeer(enode.ID{1}, "", nil), p2pNoFork, nil)
 	peerProFork = newPeer(64, p2p.NewPeer(enode.ID{2}, "", nil), p2pProFork, nil)
 
-	errc = make(chan error, 2)
-	go func() { errc <- ethNoFork.handle(peerProFork) }()
-	go func() { errc <- ethProFork.handle(peerNoFork) }()
+	errc3 := make(chan error, 2)
+	go func() { errc3 <- ethNoFork.handle(peerProFork) }()
+	go func() { errc3 <- ethProFork.handle(peerNoFork) }()
 
 	select {
-	case err := <-errc:
+	case err := <-errc3:
```

Comment:
This fix touches code that I've heavily rewritten in the snapshot-sync branch. We might want to either open the PR against my branch or wait until that's in.